src/output/plugins/RecorderOutputPlugin.cxx

```diff
@@ -56,6 +66,8 @@ class RecorderOutput final : AudioOutput {
 	 */
 	std::string format_path;
 
+	AllocatedPath archive_path = nullptr;
```

Comment:
`AllocatedPath` is initialized implicitly, unlike `Path`, which is a dumb pointer.